packages/asc-ui/src/components/ShareButton/StyledShareButton.ts

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components'
```

Comment:
Unfortunately, the way the styles are separated from the ui has no benefits while both libraries are depending on react. With the current structure is the development speed decreased because the dependent libraries must be build each tiem. asc-core would make sense when the css styling would be agnostic and would be used in an 'asc-ui' for React, for Vue, for Angular.